Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -599,7 +599,7 @@ - (UILabel *)defaultEmptyView
 
 - (void)addEmptyViewControllerToView
 {
-    if (self.emptyViewController.view.superview == nil) {
+    if (self.emptyViewController && _emptyViewController.view.superview == nil) {
```

Comment:
This method switches between `self.emptyViewController` and `_emptyViewController` a few times – could we make them all the same?

Reply:
Will do. Thanks @frosty !